formats/publication/publisher/links.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "document_collections": {
```

Comment:
These are supplied by content store / dependency resolution, not by the publishing app.